data/src/emulator.js

```diff
@@ -263,6 +263,7 @@ class EmulatorJS {
         })();
         this.canvas = this.createElement('canvas');
         this.canvas.classList.add('ejs_canvas');
+        this.videoRotation = (typeof this.config.videoRotation === "number" && [0, 1, 2, 3].includes(this.config.videoRotation)) ? this.config.videoRotation : this.preGetSetting("videoRotation") || 0;
```

Comment:
no need for the number check, the array check also implies `this.config.videoRotation` is a number